<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bug_Popout</class>
 <widget class="QWidget" name="Bug_Popout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="errGB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="title">
    <string>Word / Bit Error Rate</string>
   </property>
   <widget class="QWidget" name="errW" native="true">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>22</y>
      <width>145</width>
      <height>315</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="uvsGB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>151</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>Unregulated Voltage Supply</string>
   </property>
   <widget class="QWidget" name="uvsW" native="true">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>22</y>
      <width>145</width>
      <height>315</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="snfChk">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>30</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, the software high pass filter is enabled. Specify the filter's cutoff in Hz in the spinbox provided. This property can be modified while the acquisition is running.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Notch Filter @ 60Hz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hpfChk">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>60</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>High-Pass Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="cutoffLbl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>85</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cutoff (Hz):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="hpfSB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>130</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>log10(bit err. rate):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="berLbl">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>150</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_2">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>175</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recovered Voltage:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="recVolLbl">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>195</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="missingLbl">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>240</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_3">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>220</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Missing Frames:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Frames Found:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dataFoundLbl">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="chipIdLbl">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>330</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chip Id:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>358</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Bla bla...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1_6">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>265</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>False Frames:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="falseLbl">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>285</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="v5Lbl">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="v4Lbl">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>99</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="v3Lbl">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>177</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="v2Lbl">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>256</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="v1Lbl">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>336</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ber3Lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>177</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;span style=&quot; vertical-align:super;&quot;&gt;-3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ber4Lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>256</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;span style=&quot; vertical-align:super;&quot;&gt;-4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ber5Lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>336</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;span style=&quot; vertical-align:super;&quot;&gt;-5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ber2Lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>99</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;span style=&quot; vertical-align:super;&quot;&gt;-2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ber1Lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;span style=&quot; vertical-align:super;&quot;&gt;-1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hpfChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>cutoffLbl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hpfChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>hpfSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
